src/pastelid/pastel_key.cpp

```diff
@@ -0,0 +1,314 @@
+// Copyright (c) 2018-2021 The PASTEL-Coin developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#include "base58.h"
+#include "fs.h"
+#include "key_io.h"
+#include "pastelid/pastel_key.h"
+#include "pastelid/ed.h"
+#include "pastelid/secure_container.h"
+
+/**
+* Generate new PastelID (EdDSA448) and LegRoast public/private key pairs.
+* Create new secure container to store all items associated with PastelID.
+* 
+* \param passPhrase - secure passphrase that will be used to encrypt secure container.
+* \return pastelid_store_t map [encoded PastelID] -> [encoded LegRoast public key]
+*/
+pastelid_store_t CPastelID::CreateNewPastelKeys(SecureString&& passPhrase)
```

Comment:
use return std::tuple< string, string > instead of map

Reply:
nope, map here is for fast search of pastel ids later in the code.
Also, value right now is just string representing LegRoast public key. But later on, it can be replaced with structure containing additional fields.